doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md

```diff
@@ -0,0 +1,879 @@
+|Configure routing policy actions|sonic(config-route-map)# set as-path prepend \<list\> <br> sonic(config-route-map)# set comm-list \<name\> { add \| del } <br> sonic(config-route-map)# set community \<options\> <br> sonic(config-route-map)# set ext-community <br> sonic(config-route-map)# set ip next-hop <br> sonic(config-route-map)# set ipv6 next-hop <br> sonic(config-route-map)# set local-preference \<val\> <br> sonic(config-route-map)# set metric \<val\> <br> sonic(config-route-map)#set origin { igp \| egp \| incomplete } <br> sonic(config-route-map)# set tag \<value\> |
+
+
+#### 3.6.2.2 Show Commands
+|Command Description|CLI Command      |
+|:------------------|:-----------------|
+|Display BGP routes information |show ip bgp [vrf \<name>] { ipv4 \| ipv6 } |
+|Display summary of all BGP neighbors information |show ip bgp [vrf \<name>] { ipv4 \| ipv6 } summary|
+|Display BGP specific route information|show ip bgp [vrf \<name>] { ipv4 \| ipv6 } \<prefix> \<nbr-ip> \<path-id> |
+|Display BGP neighbor information | show ip bgp [vrf \<name>] { ipv4 \| ipv6 } neighbors [\<nbr-ip>] |
+| Display BGP neighbor received/advertised routes | show ip bgp [vrf \<name>] { ipv4 \| ipv6 } neighbors \<nbr-ip> { received-routes \| advertised-routes }|
+| Display peer-group information | show ip bgp [vrf \<name>] peer-group [\<pg-name>]|
+| Display route map information | show route-map |
+| Display IPv4 prefix list information | show ip prefix-list |
+| Display IPv6 prefix list information | show ipv6 prefix-list |
+| Display BGP community list information | show bgp community-list |
+| Display BGP extended community list information | show bgp ext-community-list |
+| Display BGP AS-Path list information | show bgp as-path-access-list |
+
+
+#### 3.6.2.3 Debug Commands
+N/A
+
+#### 3.6.2.4 IS-CLI Compliance
+NA
+### 3.6.3 REST API Support
+Various REST operations (POST/PUT/PATCH/GET/DELETE) are supported for the BGP and route map configurations.
+# 4 Flow Diagrams
+
+## 4.1 Configuration Sequence
+
+![FRR-BGP-CONFIG-SEQUENCE](images/frr-bgp-config-sequence.jpg)
+
+## 4.2 CLI Show Command Sequence
+
+### 4.2.1 CLI Show Sequence - Config information
+![FRR-BGP-CLI-SHOW-SEQUENCE1](images/frr-bgp-cli-show-sequence11.jpg)
+
+### 4.2.2 CLI Show Sequence - State/Statistics
+![FRR-BGP-CLI-SHOW-SEQUENCE2](images/frr-bgp-cli-show-sequence22.jpg)
```

Comment:
Diagram shows translib sends a synchronous message to bgpcfgd which in turn runs vtysh command. What is the messaging used between translib and bgpcfgd?

Reply:
will fix it.

Reply:
Fixed the diagram, there is no bgpcfgd involvement for state/stats show.